src/cromshell/counts/command.py

```diff
@@ -197,11 +197,14 @@ def print_call_status(call: str, indent: str, workflow_calls_metadata: dict) ->
         )
 
 
-def print_task_status_summary(workflow_metadata: dict) -> None:
+def print_task_status_summary(
```

Comment:
Added print_color parameter here mainly to help unit testing, allows me to disable coloring in testing so that its easier to evaluate the contents of the function output without dealing with the color formatting